src/SapiStatsDispatcher.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Drupal\sapi;
+
+use Drupal\Core\Config\ConfigManager;
+use Drupal\Core\Session\AccountProxy;
+use Drupal\Core\Routing\CurrentRouteMatch;
+use \Drupal\sapi\StatisticsItemInterface;
```

Comment:
`use` statement shouldn't use fully qualified name. At least the rest of the Drupal doesn't use it.